<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cframe</class>
 <widget class="QMainWindow" name="Cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1091</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Reglamentos</string>
     </attribute>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>911</width>
        <height>441</height>
       </rect>
      </property>
      <property name="text">
       <string>Este espacio virtual se utilizará para la reservación de Laboratorios de la Facultad de Ingeniería. 

Requisitos: 
 Solicitar el Laboratorios con 2 días de anticipación.
 Haber cursado las clases que involucren los equipos  presentes en el laboratorio.
Realizar la Solicitud entre 8:00 a.m. a 5:00 p.m

Reglamentos: 
 Mantener el orden y la limpieza del laboratorio solicitado para evitar amonestaciones verbales o escritas ( comité de ética).
 No ingresar comidas ni bebidas a los laboratorio de lo contrario se remitirán al comité de ética,
No modificar, Ajustar o Cambiar cualquier parte de la máquina sin autorizacion y supervisión de las personas encargados del laboratorio. 
Utilizar la vestimenta correcta: 
Zapatos Cerrados (De preferencia con punta de seguridad o cubos). 
Ropa no holgada o muy suelta. ( De preferencia jeans y camisas)
Pelo recogido . 

Notas: 
 
Una vez enviado el formulario debe esperar la confirmación de aprobación de uso del laboratorio solicitado de lo contrario no puede hacer uso del laboratorio. Cualquier pregunta o problema con su reservación puede enviar un correo a : winder.matamoros@unitec.edu.hn</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Reservar</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Laboratorio Solicitado</string>
      </property>
     </widget>
     <widget class="QComboBox" name="tipolaboratorio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Laboratorio computo #1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio computo #5 </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de Fisica</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de maquinas y herramientas </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de electronica </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de mecatronica </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de biomedica</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de energia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de industrial</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de civil</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de redes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de quimica</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de software</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de telecomunicaciones</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de automatizacion</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de computo #2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de computo #3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de computo #4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laboratorio de inteligencia artificial</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>620</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumDate">
       <date>
        <year>2023</year>
        <month>11</month>
        <day>14</day>
       </date>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Clase para la cual requiere el uso del laboratorio</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clase">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Motivo de uso del laboratorio</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="docente">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Docente</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="alumno">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Alumno</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="educacion">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Educacion continua</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="administrativo">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Administrativo</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Diariamente</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Semanalmente</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Repetir reservacion</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombre">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre completo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cuenta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero de cuenta</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="correo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Correo institucional</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="integrantes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>46</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>35</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cantidad de integrantes de grupo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombres y numero de cuentas</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>600</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha</string>
      </property>
     </widget>
     <widget class="QComboBox" name="hora_inicio">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>280</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>06 : 45</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08 : 10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09 : 55</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11 : 15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13 : 20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14 : 40</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16 : 00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17 : 20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18 : 40</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>250</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hora de inicio de la reservacion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>330</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hora de finalizacion de la reservacion</string>
      </property>
     </widget>
     <widget class="QComboBox" name="hora_final">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>370</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>08 : 10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09 : 55</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11 : 15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13 : 20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14 : 40</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16 : 00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17 : 20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18 : 40</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20  : 00</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="reservar">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>460</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Reservar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="motivo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Herramientas a usar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cuentas">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>271</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="herramientas">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>281</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Reservas Actuales</string>
     </attribute>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Reservas Actuales</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="mostrar2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>1031</width>
        <height>601</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Cargar Reservas</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
